catt/controllers.py

```diff
@@ -61,24 +61,24 @@ def all_info(self):
 ]
 
 
-def get_chromecasts() -> List[pychromecast.Chromecast]:
-    devices = pychromecast.get_chromecasts()
+def get_chromecasts_and_ips(names: Optional[List[str]] = None) -> List[Tuple[pychromecast.Chromecast, str]]:
```

Comment:
Maybe we should move these functions to a file called `discovery.py`? They seem only necessary for discovery and it would be much easier to understand the code if they were all groupped together.

Reply:
Fair point. I'll make a new PR after this is merged.